cfme/containers/pod.py

```diff
@@ -2,18 +2,19 @@
 import attr
 
 from navmazing import NavigateToAttribute, NavigateToSibling
-from widgetastic_manageiq import NestedSummaryTable, SummaryTable
 from widgetastic.utils import VersionPick, Version
 from widgetastic.widget import View
 
 from cfme.common import Taggable, TagPageView
-from cfme.containers.provider import (Labelable, ContainerObjectAllBaseView,
+from cfme.containers.provider import (ContainerObjectAllBaseView,
                                       ContainerObjectDetailsBaseView,
-                                      ContainerObjectDetailsEntities,
+                                      ContainerObjectDetailsEntities, Labelable,
                                       GetRandomInstancesMixin)
+from cfme.exceptions import ItemNotFound
 from cfme.modeling.base import BaseCollection, BaseEntity
-from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigator, navigate_to
 from cfme.utils.providers import get_crud_by_name
+from widgetastic_manageiq import NestedSummaryTable
```

Comment:
**REQUIRED**
`widgetastic_manageiq` eventually will be an external import, please put it under `widgetastic.widget`.